fitwsarah_frontend/src/views/authentication/Signup.js

```diff
@@ -1,11 +1,19 @@
+
 import React, { useState, useEffect, useCallback } from "react";
 import { useAuth0 } from '@auth0/auth0-react';
 import { Container, Spinner } from 'react-bootstrap';
 import configData from '../../config.json';
-//i have to fix sign up issue
+
+
 function AddMemberProfile() {
-    const { user, isAuthenticated, getAccessTokenSilently } = useAuth0();
+    const {user, isAuthenticated, getAccessTokenSilently} = useAuth0();
     const [accessToken, setAccessToken] = useState(null);
+    useEffect(() => {
+        if (isAuthenticated) {
+            localStorage.setItem("userData", JSON.stringify(user));
```

Comment:
Where is localstorage from?

Reply:
So local storage stores user information on the browser, i did this to try and fix the sign up bug.